plugins/package-managers/spm/src/main/kotlin/SpmModel.kt

```diff
@@ -28,6 +28,7 @@ import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.RemoteArtifact
 import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.plugins.packagemanagers.spm.SwiftPackageDependencies.Dependency
```

Comment:
To me, it seems weird to need this import for the nested class that comes from the same file. See my other comment for a proposed solution.